Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.3.md

```diff
@@ -14,34 +14,35 @@ As with all software, there is always a chance for issues and bugs, especially f
 <h2 style="font-weight:bold" align="center">New Features ✨</h2>
 
 1. Greatly improved the process of the manual testing application.
-2. Created scene management system for the testing application.
-   - Added a scene to test text rendering.
-   - Added a scene to test mouse functionality.
-   - Added a scene to test keyboard functionality.
-   - Added a scene to test graphics rendering.
-   - Created a simple scene management system to help easily add new scenes to the testing application.
-3. Created new abstract control base class named _`Velaptor.UI.ControlBase`_ for the purpose of creating custom controls.
-4. Created new _`Velaptor.UI.Label`_ control.
-5. Created new _`Velaptor.UI.Button`_ control.
-6. Created new _`Velaptor.UI.Textbox`_ control.
-   - Warning!! - This control is not finished and needs some work.
-7. Added ability to _`IContent`_ items to know if they have been created using **_Velaptor's_*** content pooling system.
-   - This ability gives the user the ability to know if a content item was created and added to the pooling system vs. created as a custom instantiation by the user.
-8. Added 2 new properties to the controls.
+2. Created scene management systems for the following testing applications:
+   - Text rendering
+   - Mouse functionality
+   - Keyboard functionality
+   - Graphics rendering
```

Comment:
Refactor to below:

**Reason:**  There is only a single testing application and that is _**VelaptorTesting**_.  The list represents the various scenes and the kind of things that each scene tests.

```suggestion
2. Created a scene management system with scenes to test the following:
   - Text rendering
   - Mouse functionality
   - Keyboard functionality
   - Graphics rendering
```